Storage.md

```diff
@@ -0,0 +1,304 @@
+
+```typescript
+let ItemsHolder = new ItemsHoldr({
+    displayChanges: {
+        Infinity: "INF";
+    }
+});
+
+ItemsHolder.addItem("limit", {
+    value: "4000",
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("limit", "Infinity");
+```
+
+`hasDisplayChange` checks to see if a value has a recorded change.
+
+```typescript
+ItemsHolder.hasDisplayChange("Infinity");
+```
+
+...and `getdisplayChange` returns the entry to replace the value with.
+
+```typescript
+let newValue: string = ItemsHolder.getDisplayChange("Infinity");
+```
+
+### Container Visibility
+
+`hideContainer` hides the container from view.
+
+```typescript
+ItemsHolder.hideContainer();
+```
+
+...while `displayContainer` makes the container visible.
+
+```typescript
+ItemsHolder.displayContainer();
+```
+
+More can be read about ItemsHoldr on its [Readme](https://github.com/FullScreenShenanigans/ItemsHoldr/blob/master/README.md).
+
+# StateHoldr
+
+StateHoldr is a module for tracking changes of items in ItemsHolder in objects called collections.
+A collection is an object whose keys are item names in ItemsHolder and whose values are key-value pairs of item attributes and values.
+These key-value pairs of attributes and values are referred to as changes.
+Collections are like a person and changes are the physical traits describing that person.
+
+## Prefix
+
+Like ItemsHoldr, StateHoldr has its own prefix property.
+All collections in StateHoldr are stored in ItemsHoldr prepended with the StateHoldr prefix.
+So the collections are stored in `localStorage` prepended with the ItemsHoldr prefix as well.
+
+```typescript
+let StateHolder = new StateHoldr({
+    ItemsHolder: new ItemsHoldr(),
+    prefix: "StateHolder"
+});
+```
+
+## Collections
+
+Collections allow for various attributes for a single item to be grouped together versus storing individual items.
+The collections the module has recorded are stored in a list keyed by `stateCollectionKeys` in ItemsHolder (no StateHoldr prefix prepended).
+StateHoldr always has one collection as its current collection.
+
+`setCollection` sets the current collection.
+
+```typescript
+// sets the collection name and the collection's value
+StateHolder.setCollection("newCollection", {
+    car: {
+        color: "red"
+    }
+});
+
+// sets only the name for the collection
+StateHolder.setCollection("newCollection");
+```
+
+If only a name is given, the collection is set to a blank object.
+
+`saveCollection` saves the current collection to ItemsHolder.
+
+```typescript
+StateHolder.saveCollection();
+```
+
+## Changes
+
+To add a change to the current collection, use `addChange`.
+`addChange` takes in the key of the item, an attribute of the item, and the value being stored.
+
+```typescript
+StateHolder.addChange("car", "color", "red");
+```
+
+If the name of the collection is known, whether or not it's the current collection, a change can be added with `addCollectionChange`.
```

Comment:
> If the name of the collection is known

Vague and unnecessary. It would be better to phrase this as changing in any collection by its name.